### ISSUE TITLE
ServicePrincipalCredentials broken for Azure Government

### ISSUE THREAD

**Issue by prburgu** [Azure/msrestazure-for-python#104]
Per announcement https://blogs.msdn.microsoft.com/azuregov/2018/04/03/azure-government-aad-authority-endpoint-update/ , the azure government AAD endpoint is changed to https://login.microsoftonline.us . However, it is still being used here https://github.com/Azure/msrestazure-for-python/blob/master/msrestazure/azure_cloud.py#L173 . I was able to work-around the issue by passing custom `auth_uri` and `token_uri`

```
ServicePrincipalCredentials(
                        cloud_environment=msrestazure.azure_cloud.AZURE_US_GOV_CLOUD,
                        client_id=clientId,
                        secret=clientSecret,
                        tenant=tenant,
                        token_uri = "https://login.microsoftonline.us/{}/oauth2/token".format(tenant),
                        auth_uri="https://login.microsoftonline.us/{}/oauth2/authorize".format(tenant)
                    )
```

**Comment by lmazuel** [comment]
@prburgu could you review PR https://github.com/Azure/msrestazure-for-python/pull/105? thanks!

**Comment by lmazuel** [comment]
Was done in CLI already: https://github.com/Azure/azure-cli/pull/4877

**Comment by lmazuel** [comment]
Fixed in #105
